sdk/resourcemanager/Azure.ResourceManager.Core/src/Azure.ResourceManager.Core.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.ResourceManager.Core</PackageId>
+    <Description>
+      Azure management core SDK for Azure resources.
+      This is a beta preview vesion.  This version uses a next-generation code generator that introduces important breaking changes, but also new features (such as intuitive authentication, custom HTTP pipeline, distributed tracing and much more).
+    </Description>
+    <PackageTags>azure;management;resource</PackageTags>
+    <NoWarn>$(NoWarn);AZC0008;AZC0001;AZC0107;CA2214;CA1036;CA1067;CA1065;SA1028</NoWarn>
```

Comment:
Lets make sure we have some comments here or issues tracking the fixing of these warning.

Reply:
I have an ADO item to follow up on removing all of those